lib/src/main/java/com/dynatrace/metric/util/MetricValues.java

```diff
@@ -34,10 +34,7 @@ private static void throwIfNaNOrInfDouble(double d) throws MetricException {
     private final long value;
     private final boolean isDelta;
 
-    LongCounterValue(long value, boolean isDelta) throws MetricException {
```

Comment:
The Javadoc for `Metric.setLongCounterValueTotal` and `.setDoubleCounterValueTotal` needs to be adapted accordingly.